lib/recurly/adjustment.ex

```diff
@@ -5,9 +5,10 @@ defmodule Recurly.Adjustment do
   for more details
   """
   use Recurly.Resource
-  alias Recurly.{Resource,Adjustment,Account,Invoice,Subscription}
+  alias Recurly.{Resource, Adjustment, Account, Invoice, Subscription}
 
   @account_endpoint "/accounts/<%= account_code %>/adjustments"
+  @invoice_endpoint "/invoices/<%= invoice_number %>/adjustments"
```

Comment:
Seems to be two streamable endpoints now. How should that be handled?

Reply:
There is probably a much cleaner way to implement, but this is how I'd imagine the interface to work:

```elixir
def stream(options) do
  keys = Keyword.keys(options)
  cond do
    :account_code in keys ->
      {account_code, options} = Keyword.pop_first(options, :account_code)
      Resource.stream(Adjustment, account_path(account_code), options)
    :invoice_number in keys ->
      {invoice_number, options} = Keyword.pop_first(options, :invoice_number)
      Resource.stream(Adjustment, invoice_path(invoice_number), options)
  end
end

# examples:
stream(account_code: "myaccount", order: :desc)
stream(invoice_number: "1001", order: :asc)
```